google/cloud/internal/debug_string_protobuf_test.cc

```diff
@@ -15,10 +15,10 @@
 #include "google/cloud/internal/debug_string_protobuf.h"
 #include "google/cloud/testing_util/scoped_log.h"
 #include "google/cloud/tracing_options.h"
+#include <google/iam/v1/policy.pb.h>
```

Comment:
Maybe using `Duration` and `Timestamp` would have been enough and we can break the dependency on IAM too?

Reply:
Well, we are never getting rid of `MinimalIamCredentialsStub`.

(But that uses `google::iam::credentials::v1`... so maybe it was foolish of me to use something from `google::iam::v1` :shrug:)

(and we have other tests that use `google::iam::admin::v1`, but those could probably be changed to `bigtable` too).